.github/workflows/ci-backend.yml

```diff
@@ -177,7 +172,7 @@ jobs:
                   touch frontend/dist/index.html
                   touch frontend/dist/layout.html
                   touch frontend/dist/shared_dashboard.html
-                  pytest ee/
+                  python -m pytest ee/ -p ee.override_clickhouse_db
```

Comment:
what's happening here? Not familiar with pytest plugins but also don't see this file anywhere